homeassistant/components/device_tracker/__init__.py

```diff
@@ -415,6 +424,7 @@ def load_config(path: str, hass: HomeAssistantType, consider_home: timedelta):
         vol.Optional(CONF_AWAY_HIDE, default=DEFAULT_AWAY_HIDE): cv.boolean,
         vol.Optional('gravatar', default=None): vol.Any(None, cv.string),
         vol.Optional('picture', default=None): vol.Any(None, cv.string),
+        vol.Optional('icon', default=None): vol.Any(None, cv.icon),
```

Comment:
If only one of these is applicable: gravatar, picture or icon, I would suggest we use `vol.Exclusive(xx, "pic")` for all three instead of vol.Optional

The "pic" forms a group for the exclusive requirement